app/controllers/content_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class ContentController < ApplicationController
```

Comment:
Just a small thing, but I wonder if it would make sense to rename this `DocumentController` as all the actions we have here are to do with Documents?  I feel like some of the difficulties with getting to grips with Whitehall centre on lack of consistency around Document vs. Edition vs. Content Item so getting consistency right early on in this app could be helpful.